docs/banzai/index.rst

```diff
@@ -2,80 +2,136 @@
-The final frame size is 4096x4096.
-Currently only 1x1 binning is supported for Sinsitros.
-
-
-Bad Pixel Masks
----------------
-Currently bad pixel masks are not included, but will be soon. Using robust statistics should guard from the effects of
-bad pixels (mostly) so we still produce clean data products.
+Stages
+------
+BANZAI is comprised of different stages, each corresponding to a single reduction step, which are chained together
+to process images. The individual stages are described below, in the order that they are executed.
 
 
 Overscan
 ========
-At LCOGT, some of our cameras have overscan regions and others that do not. Of the currently deployed instruments,
-the Sinistro cameras and the Spectral cameras have overscan regions. The SBIG cameras are capable of producing an
-overscan region but are currently configured to not produce either parallel or serial overscan.
-
-We currently use the header the keyword BIASSEC to identify the overscan region. If this keyword is set to
+We currently use the header the keyword ``BIASSEC`` to identify the overscan region. If this keyword is set to
 "Unknown", then we simply skip subtracting the overscan. We estimate a single overscan value for the whole image
 (rather than row by row).
-This is saved in the header under the keyword 'BIASLVL'. If there is no overscan section, this value is derived
+This is saved in the header under the keyword ``BIASLVL``. If there is no overscan section, this value is derived
 from the average value from the bias frames.
 
-The Sinistro frames do have overscan regions, but the overscan is currently removed by the preprocessor.
-
 
 Crosstalk
 =========
-Currently, only Sinstro images are read out using multiple amplifiers. The Sinsitro frames do have significant
+Currently, only Sinistro images are read out using multiple amplifiers. The Sinistro frames do have significant
 crosstalk between amplifiers, but this is currently removed by the preprocessor. The crosstalk is removed using
 linear coefficients that relate each quadrant to every other quadrant.
 
 
+Gain
+====
+All pixels in the frame are multiplied by the gain, using the ``GAIN`` header keyword.
+
+
 Mosaic
 ======
-Again, only the Sinstro frames currently read out with multiple amplifiers so mosaicing the amplifiers
-is only required for Sinsitros. This is currently done by the preprocessor.
+Again, only the Sinistro frames currently read out with multiple amplifiers so mosaicing the amplifiers
+is only required for Sinsitros.
 
 
 Trim
 ====
-After the overscan is subracted, the bias, dark, flat-field, and science images are trimmed
-based on the TRIMSEC header keyword.
+After the overscan is subtracted, the bias, dark, flat-field, and science images are trimmed
+based on the ``TRIMSEC`` header keyword.
+
+
+Bias Subtraction
+================
+Full frame bias images are subtracted from each of the darks, flat field images, and science frames.
```

Comment:
This is a minor annoyance but we should use the github diff width to set the maximum width of the lines before a line break in the files.